lib/data/multipleBackendGateway.js

```diff
@@ -183,6 +183,14 @@ const multipleBackendGateway = {
         return cb();
     },
 
+    isClientHandleMpu: location => {
+        const client = clients[location];
+        if (client.uploadPart) {
+            return true;
+        }
+        return false;
```

Comment:
```suggestion
        return clients[location].uploadPart;
```

Reply:
yes, an integration tests was already failing because of this

Reply:
clients[location].uploadPart is a function, we don't want to return it :) we just check to validate if the backend is manage MPU (as uploadPart function in it)

Reply:
```suggestion
        return client[location].uploadPart !== undefined;
```

Reply:
we could, but doing the all bert-e reset for a cosmetic change is maybe too much :D

Reply:
I think this is not purely cosmetic as it is checking explicitly for undefined instead of just falsy. But either way, having bert-e do more work should not dictate what does and doesn't go into PRs. 

I think there is also added value to reducing the line count as it is generally a smoother read.

Reply:
hardening the checks make sense i agree, we could even better put `typeof client[location].uploadPart === 'function'` , but i'm focusing on bugfixes here, i'll prefer to have a ticket to hardened the checks in different parts of cloudserver, not only this place (the point was not really bert-e, but the fact that i need to validate again the fix with all branches after it)

Reply:
I prefer to handle bugfixes with some hardening when possible, but the extra time to validate does not seem worth it considering the number of fixes you already have. We should push for some hardening time for this type of effort soon.

Reply:
i agree on that, hardening will be the next maintenance priority i think